test/fulfillment_v2_test.rb

```diff
@@ -0,0 +1,91 @@
+require 'test_helper'
+
+class FulfillmentV2Test < Test::Unit::TestCase
+  def setup
+    super
+    fake "fulfillment_orders/519788021", method: :get,
```

Comment:
I don't think your tests ever use this fake, removing it would be good to avoid confusion